server/neptune/temporalworker/server.go

```diff
@@ -71,7 +78,8 @@ func NewServer(config *config.Config) (*Server, error) {
 	receiverRegistry := job.NewReceiverRegistry()
 
 	// terraform job output handler
-	jobStreamHandler := job.NewStreamHandler(jobStore, receiverRegistry, config.TerraformCfg.LogFilters, config.CtxLogger)
+	jobStreamChan := make(chan *job.OutputLine)
```

Comment:
instead of doing this, can you just expose a `Close()` or `Shutdown()` method on the handler that does this?